templates/partials/navigation.phtml

```diff
@@ -19,40 +26,37 @@ declare(strict_types=1);
         <div class="collapse navbar-collapse" id="navbarCollapse">
             <ul class="navbar-nav ml-auto">
                 <li class="nav-item dropdown">
-                    <a class="nav-link dropdown-toggle<?= str_starts_with($this->url(), $this->url('about.overview')) || str_starts_with($this->url(), $this->url('security')) ? ' active' : '' ?>"
+                    <a class="nav-link dropdown-toggle<?= $currentUrl && str_starts_with($currentUrl, $this->url('about.overview')) || $currentUrl && str_starts_with($currentUrl, $this->url('security')) ? ' active' : '' ?>"
```

Comment:
With default value as string that is ` '/'` no additional checks are needed. Just replace calls to the helper with the variable: 
```suggestion
                    <a class="nav-link dropdown-toggle<?= str_starts_with($currentUrl, $this->url('about.overview')) || str_starts_with($currentUrl, $this->url('security')) ? ' active' : '' ?>"
```